needles/yelp_search_provider_results.json

```diff
@@ -0,0 +1,15 @@
+{
+  "tags": [
+    "yelp_search_provider_results"
+  ],
+  "area": [
+    {
+      "": "Whole screen, minus shell bottom bar (changes between users; clock changes)",
```

Comment:
The screenshot to go with this needle does not include a Yelp search result.

I don't think we should be matching the whole screen – I think we should be matching just the search result.

Reply:
"Search Google for ____" is not the Yelp provider.

Reply:
The perils of coming back to OpenQA after context-switching. I added that screenshot as a placeholder and never got back to it. (I know “Search Google for x” is not a Yelp search result.) Ooops :(